src/globals/scss/_css--plex-core.scss

```diff
@@ -1,52 +1,9 @@
-$font-path: 'https://unpkg.com/carbon-components@latest/src/globals/fonts' !default;
+$font-prefix: 'https://unpkg.com/carbon-components@latest/src/globals/fonts' !default;
```

Comment:
Do we want to have Carbon keep shipping Plex fonts? Or do we want to change this font path ref here as well as one in `_css—font-face.scss` to choose the right font path for Plex and the legacy font? CC @joshblack